packages/devtools-launchpad/src/index.js

```diff
@@ -31,17 +36,35 @@ if (process.env.TARGET !== "firefox-panel") {
   require("devtools-mc-assets/assets/devtools/client/themes/dark-theme.css");
 }
 
+function setThemeClassName(targetNode, newThemeClassName) {
+  // If one of the arguments is missing - bail out.
+  if (!targetNode || !newThemeClassName) {
+    return;
+  }
+
+  const {classList} = targetNode;
+
+  // Attempt to find the first theme class, assuming there's only one and
+  // ignoring class names from IGNORE_THEME_CLASS_NAMES array.
+  const activeThemeClassName = [...classList].find(cls =>
```

Comment:
nice work!  Doing a `find` might be too much. This should do the switch
```suggestion
  const activeThemeClassName = newThemeClassName == "light" ? "dark" : "light";
  // the class might almost always exist and if the class does 
  // not exist remove wont throw and error so should be fine
  classList.remove(`theme-${activeThemeClassName}`); 
  classList.add(`theme-${newThemeClassName}`)
```

Reply:
Thanks, for the feedback.
I used `find` because if somebody will add a new theme (like `camo`) a simple ternary will not work there. Not sure if we're going to support more than a few themes.

Reply:
Good point...I was working with the assumption we’ll always have light or dark, since we’ve only had those, but u r right to support future themes.
Nice work! 👍🏾